crates/re_renderer/shader/debug_overlay.wgsl

```diff
@@ -0,0 +1,55 @@
+#import <./types.wgsl>
+#import <./global_bindings.wgsl>
+
+struct UniformBuffer {
+    screen_resolution: Vec2,
+    position_in_pixel: Vec2,
+    extent_in_pixel: Vec2,
+    mode: u32,
+    _padding: u32,
+};
+@group(1) @binding(0)
+var<uniform> uniforms: UniformBuffer;
+
+@group(1) @binding(1)
+var debug_texture_float: texture_2d<f32>;
+@group(1) @binding(2)
+var debug_texture_uint: texture_2d<u32>;
+
+// Mode options, see `DebugOverlayMode`
+const ShowFloatTexture: u32 = 0u;
```

Comment:
I would prefer `SCREAMING_SNAKE_CASE` for constants, but maybe `PascalCase` is standard in wgsl?

Reply:
I would as well, but then I made it an enum on the rust side and clippy got angry with me, so I took that